mabwiser/mab.py

```diff
@@ -189,10 +190,32 @@ def _validate(self):
             if self.arm_to_scaler is not None:
                 check_true(isinstance(self.arm_to_scaler, dict), TypeError("Arm_to_scaler must be a dictionary"))
 
+    class Popularity(NamedTuple):
+        """Randomized Popularity Learning Policy.
+
+        Returns a randomized popular arm for each prediction.
```

Comment:
This should include the formula for calculating the distribution

Reply:
I didn't see one for Greedy so I skipped it, but that's a good point

Reply:
For greedy it's just calculating the means, but here we are generating a distribution

Reply:
done